Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/Merchant.yaml

```diff
@@ -630,6 +630,15 @@ DriverIntelligentPoolConfig:
 TransporterConfig:
   derives: Generic,Show,'UsageSafety
   types:
+    CancellationRateBasedNudgingAndBlockingConfig:
```

Comment:
Keeping configs as json is not a good idea inside the table. It will become very difficult to handle. 
We should think of some better approach, for now its better we you can flatten them at beam side if not at domain side.

Reply:
Sure, did change for it